pkg/api/types.go

```diff
@@ -1518,6 +1518,9 @@ type PodSecurityContext struct {
 	//
 	// If unset, the Kubelet will not modify the ownership and permissions of any volume.
 	FSGroup *int64 `json:"fsGroup,omitempty"`
+	// The seccomp profile that containers in the pod will run with.  Any container
+	// security context not defining its own profile will inherit the pod level profile.
+	SeccompProfile *string `json:"seccompProfile,omitempty"`
```

Comment:
Why `*string` and not `string`? Both `nil` and empty string mean the same, don't they?